docs/usage-guide.md

```diff
@@ -93,9 +93,11 @@ await client.reset() # Empties and completely resets the database. ⚠️ This i
 <Tabs queryString groupId="lang" className="hideTabSwitcher">
 <TabItem value="py" label="Python">
 
-Chroma can also be configured to use an on-disk database, useful for larger data which doesn't fit in memory. To run Chroma in client server mode, run the docker container:
+Chroma can also be configured to use an on-disk database, useful for larger data which doesn't fit in memory. To run Chroma in client server mode:
```

Comment:
I think there are other references to docker we need to rethink as well. For example in auth - https://docs.trychroma.com/usage-guide#basic-authentication